lib/commons/forms/index.js

```diff
@@ -1,8 +1,23 @@
+// TODO: es-module-commons. convert to:
+// export { default as isAriaCombobox } from 'path'
+import isAriaCombobox from './is-aria-combobox';
+import isAriaListbox from './is-aria-listbox';
+import isAriaRange from './is-aria-range';
+import isAriaTextbox from './is-aria-textbox';
+import isNativeSelect from './is-native-select';
+import isNativeTextbox from './is-native-textbox';
+
 /**
  * Namespace for forms-related utilities.
  * @namespace commons.forms
  * @memberof axe
  */
-
-var forms = {};
+const forms = {
```

Comment:
This was probably discussed in the proposal and I'm just not remembering the answer, but why aren't we `export`ing the `forms` object?

Should this be:

```
export { isAriaCombobox, isAriaListbox, isAriaRange, ... }

// or:

export default forms
```

Reply:
I don't think I specifically mentioned it, but I figured we would do

```js
export { default as isAriaCombobox } from './is-aria-combobox.js'
export { default as isAriaListbox } from './is-aria-listbox.js'
```

Then the main commons `index.js` file would group these as "forms"

Reply:
Should we make this change now?

Reply:
We can't yet. I would need all the common subdirectors converted first so I can add them to the main commons object. The way we concat now is that concat the commons index file first and then every file adds their function to it. So I can't webpack the commons index file yet as that would break the code that expects commons to be a global variable.

Reply:
Ahh, ok, that makes sense. Thanks for clarifying.